### PR TITLE
person A changes

### PR DESCRIPTION
**Pull request by spv-hub**

### CODE REVIEW

File: favorite-color.text [hunk 1]
```diff
@@ -0,0 +1 @@
+Some text from person A
```
Comment (seattlefurby17) [inline favorite-color.text:1]: Sandy, what a line of code!

**Review by seattlefurby17 — APPROVED** [review]

File: favorite-color.text (hunk 1 above)
Comment (beccaelenzil) [inline favorite-color.text:1]:
```suggestion
Some text from person A, by Sandy
```

**Review by beccaelenzil — CHANGES_REQUESTED** [review]

File: favorite-color.text [hunk 2]
```diff
@@ -0,0 +1 @@
+Some text from person A, by Sandy
```
Comment (beccaelenzil) [inline favorite-color.text:1]:
```suggestion
Some text from person A, by Sandy, hurray!
```

Comment (beccaelenzil) [inline favorite-color.text:1]:
```suggestion
Some text from person A, by Sandy, hurray hurray!
```

**Pull request merged**